### ISSUE TITLE
Override theme

### ISSUE THREAD

**Issue by PeterFred** [gusparis/react-native-month-year-picker#149]
Currently the theme is set from system settings.
Our app can set a dark theme independently of the system settings.
It would be good to be able to overwrite the theme input.

**Comment by gusparis** [comment]
did you check the `autoTheme` prop already?

**Comment by PeterFred** [comment]
Yes, that uses the phones system theme. I want to manually set it, ie override the autoTheme prop.
Eg, phone is in lightMode, I can set the picker to be in darkMode
